### PR TITLE
tmuxPlugins.dracula: unstable-2020-12-28 -> unstable-2021-02-09

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Motivation for this change
bump the package

###### Things done

change rev and sha

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112608` at 1164fb4a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/07f172703b022b73e23489b51ea71c71">tmuxPlugins.dracula</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/misc/tmux-plugins/default.nix:34:14:
```
   |
34 |       inherit pluginName unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`configurePhase` should probably contain `runHook preConfigure` and `runHook postConfigure`.

Near pkgs/misc/tmux-plugins/default.nix:34:14:
```
   |
34 |       inherit pluginName unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112608` at 1164fb4a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/27696dbd63437a6f4259041071b6d0f5">tmuxPlugins.dracula</a></li></li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild` and `runHook postBuild`.

Near pkgs/misc/tmux-plugins/default.nix:34:14:
```
   |
34 |       inherit pluginName unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`configurePhase` should probably contain `runHook preConfigure` and `runHook postConfigure`.

Near pkgs/misc/tmux-plugins/default.nix:34:14:
```
   |
34 |       inherit pluginName unpackPhase configurePhase buildPhase addonInfo preInstall postInstall;
   |              ^
```
</li>
  </ul>
</details>

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -129,12 +129,12 @@ in rec {
 
   dracula = mkDerivation rec {
     pluginName = "dracula";
-    version = "unstable-2020-12-28";
+    version = "unstable-2021-2-09";
```
Comment (SuperSandro2000) [inline pkgs/misc/tmux-plugins/default.nix:132]:
```suggestion
    version = "unstable-2021-02-09";
```

**Comment by ethancedwards8** [comment]
Fixed @SuperSandro2000

**Pull request merged**